webrtc.html

```diff
@@ -8852,6 +8854,34 @@ <h2 id="sec.stats-model">Statistics Model</h2>
       [[!WEBRTC-STATS]].</p>
     </section>
     <section>
+      <h3>The stats selection algorithm</h3>
+      <p>
+	The <dfn>stats selection algorithm</dfn> is as follows:
+	<ul>
+	  <li>If the selector is a MediaStreamTrack, return the following objects:
+	    <ul>
+	      <li>
+		The RTCMediaStreamStats corresponding to the selector.
+	      </li>
+	      <li>
+		If the selector track is inbound, the RTCInboundRTPStreamStats corresponding to the selector.
+	      </li>
+	      <li>
+		If the selector track is outbound, the RTCOutboundRTPStreamStats corresponding to the selector.
+	      </li>
+	      <li>
+		At the implementation's discretion, other stats that the
+		implementation considers relevant.
```

Comment:
My reason for leaving it in is that we have a generic permission to add new stats (implementation specific or newly standardized), so preventing new stats from being added seems awkward.
If we want to prevent the addition of existing objects, such as candidates and transports, we should say so.
We could also replace this with something like:
NOTE: New versions of this specification may add more stats to this list, so the caller SHOULD be prepared to handle unexpected stats objects.